src/components/Projects.tsx

```diff
@@ -37,18 +51,17 @@ export default function Projects() {
             <div id="Projects" className="border-light-brown bg-dark text-white px-20 pt-24 lg:px-40">
             <div className="sm:flex sm:justify-between sm:items-baseline">
                 <div className="sm:w-0 sm:flex-1">
-                <h1 id="message-heading" className="text-2xl font-medium">
-                    Full-Stack <span className="text-light-brown">Developer</span>
+                <h1 id="message-heading" className="text-3xl font-medium">
+                    Personal <span className="text-light-brown">Portfolio</span>
                 </h1>
-                <p className="mt-1 text-sm text-gray-500 truncate">Checkout and Payments Team</p>
+                <p className="mt-1 text-lg text-gray-500 truncate">You can check some of my Github project repositories below:</p>
```

Comment:
Github it's just a platform. 
In this case, seems the content of the project is something related to Github. So I think it's better to use just  project.

```suggestion
                <p className="mt-1 text-lg text-gray-500 truncate">You can check some of my project repositories below:</p>
```